third_party/amd/lib/TritonAMDGPUTransforms/StreamPipeline.cpp

```diff
@@ -71,7 +71,7 @@ class LoopPipeliner {
   /// shared mem and a next buffer stored in regs.
   int numStages = 2;
 
-  /// Arg indicies
+  /// Arg indicies in in pplForOp
```

Comment:
Typo? I don't quite get what pplForOp stands for, is it the new ForOp?

Reply:
the "ppl" in "pplForOp" stands for "pipeline" - pplForOp is a member variable. While reading this code I was confused about whether this referred to indices in the "forOp" op, or the "pplForOp" op, so I added this for future readers of this code.

https://github.com/triton-lang/triton/blob/main/third_party/amd/lib/TritonAMDGPUTransforms/StreamPipeline.cpp#L51-L52